tests/core/test_db.py

```diff
@@ -32,14 +44,27 @@ def test_init_database_none(patch_settings: Settings):
 
     init_db()
 
-    expected = [
-        (select(User).where(User.name == "Robbert"),),
-        (select(Status).where(Status.name == "Todo"),),
-        (select(Task).where(Task.title == "First task"),),
-    ]
+    for i, call_args in enumerate(Session.exec.call_args_list):
```

Comment:
it probably better to use the pytest.parametrize option to handle multiple test cases with the same logic. 

See: https://docs.pytest.org/en/7.0.x/how-to/parametrize.html

Reply:
I think I did that now. Can you check @berrydenhartog ?